src/components/MultipleChoiceQuestion.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { PropTypes } from 'react';
+import PureComponent from 'react-pure-render/component';
+import Immutable from 'immutable';
+
+import Dropdown from 'components/Dropdown';
+
+export default class MultipleChoiceQuestion extends PureComponent {
+  static propTypes = {
+    id: PropTypes.number.isRequired,
+    question: PropTypes.instanceOf(Immutable.Map).isRequired,
+    nextQuestionOptions: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const { id, question, nextQuestionOptions } = this.props;
+    const defaultNextQuestionId = question.get('default_next_question_id');
```

Comment:
What's the difference between `next_question_id` and `default_next_question_id` ?
Can't we initialize `next_question_id` to a default value instead of making it another variable ?